.github/workflows/ci.yml

```diff
@@ -28,7 +28,15 @@ jobs:
         run: poetry run python manage.py migrate --settings=config.settings.local
       - name: fill database
         run: poetry run python manage.py loaddata --settings=config.settings.local e2e_tests/database/test_database.json
-      - name: build application
-        run: docker compose up -d --build
+      - name: build testing application
+        run: |
+          sed -i 's/ENVIRONMENT_NAME=local/ENVIRONMENT_NAME=testing/g' .env
+          docker-compose -f docker-compose.yml -p testing up -d
+      - name: start reverse-proxy
+        run: |
+          cd docker/reverseproxy
+          docker network create -d bridge production_nginx_network
+          export CERTIFICATES_PATH=./ssl_certificates_localhost
+          docker compose up -d --build
```

Comment:
The way the deployed e2e test must be executed changed drastically here, but not in the README. Using the old instructions there, the tests fail for me. Also, `export` cannot be used when developing on Windows so just copying over these instructions from ci.yml to the README won't help either.
We'll have to think of a way to execute tests locally that works for all systems, maybe using a specific .env file for the reverse proxy that has the right CERTIFICATES_PATH set already, combined with a different .env file for the actual app that sets the ENVIRONMENT_NAME.
I'll try to work on that tomorrow.

Reply:
I moved the certs path setting and the "testing" env name and compose project name (`-p testing`) to the compose .env files to allow local and CI tests to use the same commands, which I then updated in ci.yml and README.md.